packages/core/testing/src/r3_test_bed_compiler.ts

```diff
@@ -26,6 +27,17 @@ function isTestingModuleOverride(value: unknown): value is TestingModuleOverride
       value === TestingModuleOverride.OVERRIDE_TEMPLATE;
 }
 
+function assertNoStandaloneComponents(
+    types: Type<any>[], resolver: Resolver<any>, location: string) {
+  types.forEach(type => {
+    const component = resolver.resolve(type);
+    if (component && component.standalone) {
+      throw new Error(generateStandaloneInDeclarationsError(
+          type, 'of the "TestBed.configureTestingModule" call'));
+    }
+  });
+}
```

Comment:
I believe we just want to pass the `location` into an error message generator fn?
```suggestion
function assertNoStandaloneComponents(
    types: Type<any>[], resolver: Resolver<any>, location: string) {
  types.forEach(type => {
    const component = resolver.resolve(type);
    if (component && component.standalone) {
      throw new Error(generateStandaloneInDeclarationsError(type, location);
    }
  });
}
```

Reply:
yeah sorry, again a refactoring mistake :sweat: